desktop/src/app/components/navbar/warnings/modals/delete-outliers-modal.component.ts

```diff
@@ -47,6 +60,25 @@ export class DeleteOutliersModalComponent {
             : '<code>' + this.field.name + '</code>';
     }
 
+    public async prepareDeleteAll() {
+        this.deleteAll = !this.deleteAll;
```

Comment:
I think this line should be moved to a separate function "toggleDeleteAll". The other code of this function does not have to be called when deleteAll is set to false.

Reply:
I think I addressed this now, but it would be great if you could check again, as I am not sure about the style of the if-condition in toggleDelete/ReplaceAll().

Reply:
If statements can be reduced to a single line if it fits, and you can omit the "=== true" part in case of a boolean check. So the lines 63-65 can be replaced with:

if (this.deleteAll) this.prepareDeleteAll();

prepareDeleteAll() and prepareReplaceAll() can also be made private if they are not called from the HTML template.